hooks/research-stop-hook.sh

```
@@ -0,0 +1,264 @@
+
+# Research Stop Hook
+# Prevents session exit when research hasn't met target source count
+# Reports resource usage on completion
+
+set -euo pipefail
+
+# Fail-open if jq is not available
+command -v jq >/dev/null 2>&1 || { exit 0; }
+
+# Read hook input from stdin
+HOOK_INPUT=$(cat)
```

Shellcheck (0.11.0)

[warning] HOOK_INPUT appears unused. Verify use (or export if used externally).

(SC2034)